app/assets/javascripts/items.js

```diff
@@ -22,18 +25,40 @@ function loadAllItems(container) {
   })
 }
 
+let merchName = function (item_id) {
+  fetch(`http://localhost:3000/api/v1/items/${item_id}/merchant`)
```

Comment:
This method will use fetch to hit an API then set the desired information to the variable `merchantName` to then be put into the view